stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1597 @@
+{
```

Comment:
<p>I think the info on the UK pop and the priority groups should be in the high level, not technical, narrative, so we might need to think about how the high level flows and if this needs to be moved around at all (if you don't read the technical stuff looking at the first and last row, you are missing some context on this data).</p><p> </p><p>But I also wanted to follow up on one statement above: "we can see that first doses for groups 1-9 have been completed as well as second doses for groups 1-4." We can see that the # of people = to the combined pop of those groups has been vaccinated, but do we know that those groups have reached 100% of the first and second doses? Looks like there is some vaccine hesitancy in some of these priority groups, so might be worth cross-referencing https://www.england.nhs.uk/statistics/statistical-work-areas/covid-19-vaccinations/. Hopefully this isn't a can of worms - maybe you just need a caveat, and to link to this other analysis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Good point about the priority group "completion" dates. I've added a paragraph to explain the caveats with this and weakened statements about populations being vaccinated (vs. the sub-totals being helpful indicators of how much progress has been made so far).

Reply:
<p>I've also reworked the ordering here so interacting with the data comes first followed by the description, everything before the paragraph starting "Although the first Covid-19 vaccine..." can be in  the technical narrative, then high-level narrative until the line starting "Bearing all the above in mind,..."</p>